react-app/src/ExerciseOverview/ExerciseDetail.js

```diff
@@ -60,12 +70,16 @@ export default function ExerciseDetail({ item, totalSteps, setSelectedStep, setS
         <Text style={styles.description}>{description}</Text>
         <View style={styles.instructionContainer}>
           <FlatList
-            contentContainerStyle={{ paddingBottom: 30 }}
             data={instruction}
             renderItem={({ item }) => <Text style={styles.instruction}>{item}</Text>}
           />
         </View>
       </View>
+      {path !== 'Exercise Steps' && (
```

Comment:
Can we inject it as boolean to ExerciseDetails instead of checking path?

Reply:
I agree, because we might have other exercises in the future.
For now there are 3 things like `path === 'Exercise Steps'` in this page. I think we can consider to put the logical operation in the beginning and return a boolean. Then change layout by that boolean after that.

Reply:
Thank you guys! I will fix this